### DISCUSSION TITLE
When changing the forecasting horizon, the outcome is the same, why!?

### DISCUSSION THREAD

**Discussion by Msaleh87** [ourownstory/neural_prophet#1403] [Q&A - get help using NeuralProphet]
So I have a year's worth of data with a 5-min resolution. I am trying to predict the next 6 hours, and the next 48 hours. Here is the training model I am using:
```
m = NeuralProphet(n_forecasts = prediction_horizon, 
    n_lags=int(14*24* timestep), #<--------- from 16 to 7
    yearly_seasonality="auto",
    weekly_seasonality="auto",
    daily_seasonality="auto", 
    growth="off", 
    seasonality_reg= 0.5, 
    optimizer = 'AdamW',
    learning_rate = 0.001, 
    n_changepoints="auto", 
    changepoints_range = "auto"
                )

m = m.add_country_holidays("US", mode="additive") # I added an equal sign, check how would this impact the original code
metrics = m.fit(N_Ph_Train , freq = freq)   # N_Ph_data_train_resample_10min[:Threshold_10] 
m.highlight_nth_step_ahead_of_each_forecast(step_number=m.n_forecasts)
```

When I did the prediction for the next 6 hours, it gave me the exact first 6 hours of the 48 hours forecast horizon! Why? I was expecting that if I change the forecast horizon that the accuracy would change.

**Comment by leoniewgnr** [comment]
Hi @Msaleh87, I'm not super sure if I understood your question. Would you mind explaining me again?
